### ISSUE TITLE
物品包功能，用于一个格子允许多个物品

### ISSUE THREAD

**Issue by YufiriaMazenta** [YufiriaMazenta/Craftorithm#45]

**Comment by ForgenDT** [comment]
Hello, I would like to know if this issues corresponds to this:
be able to craft an item with more than one item per slot.
Google Translate doesn't always translate very well from Chinese to English ._.

I tested a lot of craft plugins but only one supports this feature.
the plugin in question is not as practical as yours, in addition to having 2 dependencies...

in my case this is the only thing stopping me from using your plugin.
In any case, it's very practical to use :D (your plugin)

**Comment by YufiriaMazenta** [comment]
> Hello, I would like to know if this issues corresponds to this: be able to craft an item with more than one item per slot. Google Translate doesn't always translate very well from Chinese to English ._.
> 
> I tested a lot of craft plugins but only one supports this feature. the plugin in question is not as practical as yours, in addition to having 2 dependencies...
> 
> in my case this is the only thing stopping me from using your plugin. In any case, it's very practical to use :D (your plugin)

No, this feature means that in one slot, multiple types of items are allowed to be used, similar to how all wooden planks can be used for crafting chests

**Comment by YufiriaMazenta** [comment]
However, the features you mentioned are also in the update plan and will be added in the future
